google/cloud/spanner/testing/random_instance_name.h

```diff
@@ -24,8 +24,8 @@ namespace cloud {
 namespace spanner_testing {
 GOOGLE_CLOUD_CPP_INLINE_NAMESPACE_BEGIN
 
-/// Create a random instance name given a PRNG generator.
-std::string RandomInstanceName(google::cloud::internal::DefaultPRNG& generator);
+/// Returns a random instance ID given a PRNG generator.
+std::string RandomInstanceName(internal::DefaultPRNG&);
```

Comment:
do you want to change this to `RandomInstanceId` ?

Reply:
I do want to, yes.  (You may have noticed some other name changes I've been making to try to reduce confusion.)  However, in this case I decided that changing the function/file name wasn't worth it.  I might revisit that choice later though.